ci/build-push-multiplatform.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+docker buildx build --push --platform=linux/amd64     --build-arg FLUENTD_FLAVOR=debian       --build-arg BUILD_TAG="${BUILD_TAG}" --tag "${REPO_URL}:${BUILD_TAG}-amd64" . & amd64_pid=$!
```

Comment:
We should check for requirements IMHO

Reply:
@sumo-drosiek I've added this check, please see if it's OK https://github.com/SumoLogic/sumologic-kubernetes-fluentd/pull/187/commits/2f03be69bd11b76bb10e0effeb1cb759258410cb.